static/plume-front.js

```diff
@@ -0,0 +1,572 @@
+"use strict";
```

Comment:
I think this file souldn't be committed, it's auto generated and things like `__cargo_web_snippet_199d5eb25dfe761687bcd487578eb7e636bd9650` are probably not reproducible between builds